### PR TITLE
updated dockerfile

### PR DESCRIPTION
**Pull request by epasham**
The dockerfile is not working. updated the steps and dependencies to get build created.
connection test was also failing as the code is expecting running mongodb instance. needed to bypass the connection test script to complete the build

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,16 @@
-FROM        quay.io/prometheus/busybox:latest
-MAINTAINER  Alexey Palazhchenko <alexey.palazhchenko@percona.com>
+FROM golang:1.11 as builder
 
-COPY mongodb_exporter /bin/mongodb_exporter
+MAINTAINER Ekambaram Pasham <ekambaram_pasham@infosys.com>
```
Comment (AlekSi) [inline Dockerfile]: Do you really intend to maintain it?

**Comment by AlekSi** [comment]
> connection test was also failing as the code is expecting running mongodb instance.

CONTRIBUTING.md mentions this. We use Docker Compose for integrational testing.

File: Dockerfile (hunk 1 above)
Comment (epasham) [reply Dockerfile]: i can maintain the build and deployment configuration

**Comment by DexterHD** [comment]
Hi @epasham . Could you fix Codacy checks described here: https://app.codacy.com/app/Percona/mongodb_exporter/pullRequest?prid=2148824 ?

**Comment by DexterHD** [comment]
I'm closing it. Because it's same as #112 and this Dockerfile doesn't work.

**Pull request closed without merging**